src/app/ui/dialog/add-quest-dialog/add-quest-dialog.component.ts

```diff
@@ -22,12 +22,12 @@ const textValidator: ValidatorFn = (control: AbstractControl): ValidationErrors
   }
 
   const text = (control.value || '').trim();
-  const withinCharLimits = text.length > 2 && text.length <= 50;
+  const withinCharLimits = text.length > 2 && text.length <= 100;
   if (!withinCharLimits) {
     return { invalidText: true };
   }
 
-  const hasInvalidChars = !text.match(/^[\w ]*$/);
+  const hasInvalidChars = !text.match(/^.{3}.*$/);
```

Comment:
this is no different from `.*` that matches any given input. Suggesting this below instead, which means that as long as it have a minimum of 3 consecutive alphanumeric characters then it's valid.
```suggestion
  const hasInvalidChars = !text.match(/\w{3,}/);
```

Reply:
Whoops, you're right.